main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "github.com/19chonm/461_1_23/cli/commands"
```

Comment:
Ooh nice job with the separate packages. I'm still a bit unsure as to how we should split up my code into packages, so I'd appreciate your opinion on it.